sdk/notificationhubs/notification-hubs/api-extractor.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "mainEntryPointFilePath": "types/src/index.d.ts",
+  "mainEntryPointFilePath": "types/src/index.docs.d.ts",
```

Comment:
This will be done programmatically, so this diff won't be part of the resulting PR